test/backend/common.py

```diff
@@ -132,6 +139,7 @@ def compile_model(model, emit):
             "--constants-to-file-total-threshold="
             + str(args.constants_to_file_total_threshold)
         )
+    command_list += get_compile_option(name + "_cpu")
```

Comment:
I would have a `if` directly here, it's easy to follow and faster:
```python
if args.dynamic and test_name in variables.test_to_enable_dimparams_dict:
        command_list.append("--dimParams=" + variables.test_to_enable_dimparams_dict[test_name])
```

Reply:
Fixed. Thanks!